controllers/tab/issueEntry/form/IssueEntryPublicationMetadataForm.inc.php

```diff
@@ -342,6 +344,20 @@ function execute($request) {
 				$submission->setCopyrightHolder(null, null);
 				$submission->setLicenseURL(null);
 			}
+			
+			$submission->setDatePublished($this->getData('datePublished'));
+			
+			$submissionDao = Application::getSubmissionDAO();
+			if ($request->getUserVar('submissionRevision')) {
+				$revision = (int)$request->getUserVar('submissionRevision');
```

Comment:
I am not sure if it would be OK/better to consider those new checkboxes as part of the form, like all the other form fields?